indico/util/string.py

```diff
@@ -190,7 +191,7 @@ def remove_non_alpha(text):
 
 
 def str_to_ascii(text):
-    return translitcodec.long_encode(text)[0].encode('ascii', 'ignore').decode().strip()
+    return unidecode.unidecode(text).encode('ascii', 'ignore').decode().strip()
```

Comment:
Like this we keep the better-quality output from `translitcodec` and only use `unidecode` as a fallback for anything it could not handle.

```suggestion
    text = translitcodec.long_encode(text)[0]
    text = unidecode.unidecode(text)
    # theis ascii roundtrip SHOULD not be needed, but let's stay safe just in case...
    return text.encode('ascii', 'ignore').decode().strip()
```